web/src/index.html

```diff
@@ -10,6 +10,35 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,500" rel="stylesheet">
+  <style>
+    .mtk1 { color: #000000; }
```

Comment:
This is needed to give the color to the tokens in the colorized code.

Reply:
Can't we use the default one ? I am not sure which highlighter js library we're using...